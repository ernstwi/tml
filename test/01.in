init foo
bar { x < 10 }

foo -> bar
    guard { x >= 5 }
bar -> baz
