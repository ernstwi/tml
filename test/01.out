{
    "edges": [
        {
            "from": "bar",
            "guard": null,
            "reset": null,
            "sync": null,
            "to": "baz"
        },
        {
            "from": "foo",
            "guard": "x>=5",
            "reset": null,
            "sync": null,
            "to": "bar"
        }
    ],
    "locations": [
        {
            "id": "bar",
            "initial": false,
            "invariant": "x<10"
        },
        {
            "id": "foo",
            "initial": true,
            "invariant": null
        }
    ]
}
